app/index.js

```diff
@@ -218,5 +191,12 @@ module.exports = generators.Base.extend({
         self.props
       );
     });
+  },
+
+  end: function () {
+    if(!this.options.skipInstall) {
+      var done = this.async();
+      this.spawnCommand('npm', ['install']).on('close', done);
```

Comment:
we'll want to use loglevel error or it will be noisey.  This is how it's done in donejs-cli: https://github.com/donejs/cli/commit/e00bdc1d83d8e0afb9edc570ee0386d5dc3ffced